src/Results/ResultCollection.php

```diff
@@ -12,8 +13,40 @@ class ResultCollection extends Collection
      */
     public function orderByScoreDesc(): self
     {
-        return $this->sortByDesc(function ($result) {
+        return $this->sortByDesc(function (Result $result) {
             return $result->getScore();
         });
     }
+
+    /**
+     * Normalize results against config.
+     * @param  Config $config
+     * @return array
+     */
+    public function normalizeAgainst(Config $config): array
```

Comment:
Could `normalizeAgainst` still return `self` here? I think if this method returned an array it might lead to confusion.

A separate method within this class `toArray()` I think would be pefect. 

Then in the main `ChurnCommand` you could have this when adding the Results to the table?

```
$table->addRows($results->toArray());
```

What do you think?

Reply:
Ye you're right.